nssp_demo/collate_plots.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python3
+
+import argparse
+import fnmatch
+import logging
+import os
+from pathlib import Path
+
+from pypdf import PdfWriter
+from utils import get_all_forecast_dirs
+
+
+def write_merged_pdf(
```

Comment:
Current name suggest a merged_pdf is supplied as an argument and then written, but this function actually merges a list of pdf files. Suggest renaming.

Reply:
Function name changed in to `merge_pdfs_and_save` in af76cfd